src/lib/pickles/verify.ml

```diff
@@ -166,66 +166,66 @@ let verify_heterogenous (ts : Instance.t list) =
         plonk )
   in
   let open Backend.Tock.Proof in
+  let open Async in
+  let%bind accumulator_check =
+    Ipa.Step.accumulator_check
+      (List.map ts ~f:(fun (T (_, _, _, _, T t)) ->
+           ( t.statement.proof_state.me_only.sg
+           , Ipa.Step.compute_challenges
+               t.statement.proof_state.deferred_values.bulletproof_challenges
+           ) ))
+  in
   Common.time "batch_step_dlog_check" (fun () ->
```

Comment:
This (and the other) timing function calls now have a slightly decreased scope in what they are timing now. For example, this function call used to measure the call to `Ipa.Step.accumulator_check` as well. Now that this is all deferred, we probably need a `Common.time_deferred` helper function if we want to keep the same timing logic.